src/EA.Weee.Web/Areas/Admin/Views/ManageEvidenceNotes/ViewAllEvidenceNotes.cshtml

```diff
@@ -56,22 +56,22 @@
                             <caption class="govuk-table__caption govuk-visually-hidden">All evidence notes</caption>
                             <thead class="govuk-table__head">
                                 <tr class="govuk-table__row">
-                                    <th class="govuk-table__header" scope="col">
+                                    <th class="govuk-table__header" scope="col" style="vertical-align: bottom; width: 10%;">
                                         Reference ID
                                     </th>
-                                    <th class="govuk-table__header" scope="col">
+                                    <th class="govuk-table__header" scope="col" style="vertical-align: bottom; width: 15%;">
```

Comment:
@GuidoWorkDev I dont think we need this vertical-align: bottom; style...

Reply:
removed